tests/lax_numpy_test.py

```diff
@@ -4254,6 +4254,20 @@ def testArgsortUnstable(self, dtype, shape, axis, descending):
       actual = jnp.take_along_axis(x, indices, axis=-1 if axis is NO_VALUE else axis)
     self.assertArraysEqual(actual, expected)
 
+  def _assertSamePartionedArrays(self, jnp_output, np_output, axis, kth, shape):
```

Comment:
I refactored the assertion part between testPartition and testArgpartition functions and reused this method in added tests. If there can be another way of doing the tests I'm happy to implement it